MilitaryManager/client-app/src/app/modules/attachments-module/pages/decree-new/decree-new.component.ts

```diff
@@ -0,0 +1,48 @@
+import { Component, OnInit, ViewChildren, ViewContainerRef } from '@angular/core';
+import { Test1Component } from 'src/app/components/test1/test1.component';
+import { ApiService } from 'src/app/shared/services/api/api.service';
+
+@Component({
+  selector: 'app-decree-new',
+  templateUrl: './decree-new.component.html',
+  styleUrls: ['./decree-new.component.scss']
+})
+export class DecreeNewComponent implements OnInit {
+  
+  //@ts-ignore
```

Comment:
Consider configuring null checks in **tsconfig.json** instead of putting `//@ts-ignore` above each line.